api.go

```diff
@@ -474,45 +474,45 @@ func DeleteContainer(podID, containerID string) (*Container, error) {
 
 // StartContainer is the virtcontainers container starting entry point.
 // StartContainer starts an already created container.
-func StartContainer(podID, containerID string) (*Container, error) {
+func StartContainer(podID, containerID string) (*Pod, *Container, error) {
```

Comment:
If we create a new public function `PodFromID()`, we will have to fetch the pod from this method (which has a performance cost), while we already have this pod from inside the `StartContainer()`.
I think it duplicates things, doesn't it ?